recipes-bsp/partition/qcom-partition-confs_1.0.bb

```diff
@@ -27,6 +27,15 @@ PARTCONF ?= ""
 PARTCONF:qcm6490 ?= "qcm6490-partitions.conf"
 PARTCONF:qcs9100 ?= "qcs9100-partitions.conf"
 
+# For machines with a published cdt file, let's make sure we flash it
```

Comment:
Could you please also expand on what CDT i  and what is contains in the  `qcom-partition-confs: add support for CDT" commit message. Thanks!

Reply:
Sure. I can add more information. I will update the PR.
CDT is Configuration Data Table. It contains the platform and subtype IDs, the same explained here https://android.googlesource.com/kernel/msm.git/+/android-msm-bullhead-3.10-n-preview-1/Documentation/devicetree/bindings/arm/msm/board-id.txt. 
The IDs are 'flashed', at runtime they are read by the bootloader, and it will search through all the DTBs on the system for the one that 'matches' looking at the board-id in the DTB.

That solution was never accepted upstream, and used mostly in Android. We are working on a 'new' solution for qualcomm linux.